module-infrastructure/persistence-db/src/main/kotlin/site/yourevents/invitation/repository/InvitationRepository.kt

```diff
@@ -26,4 +28,19 @@ class InvitationRepository(
         return invitationJPARepository.findById(id)
             .getOrNull()?.toDomain()
     }
+
+    override fun findByMember(member: Member): List<Invitation> {
+        return MemberEntity.from(member)
+            .let { memberEntity ->
+                invitationJPARepository.findByMember(memberEntity)
+                    .map { it.toDomain() }
+            }
+    }
+
+    override fun countByMember(member: Member): Int {
+        return MemberEntity.from(member)
+            .let { memberEntity ->
+                invitationJPARepository.countByMember(memberEntity)
+            }
+    }
```

Comment:
let을 사용함으로써 로직의 가독성이 조금 떨어진다는 생각입니다..! let을 사용하지 않고도 코드를 더욱 간결히 표현해줄 수 있다고 생각합니다! 👍 
```suggestion
    override fun findByMember(member: Member): List<Invitation> =
        invitationJPARepository.findByMember(MemberEntity.from(member))
            .map { it.toDomain() }

    override fun countByMember(member: Member): Int =
        invitationJPARepository.countByMember(
            MemberEntity.from(member))
```

Reply:
오옹 그렇네요!
아래와 같이 메서드 참조 방식으로 변경하면 더 간결하지 않을까요?
```kotlin
override fun findByMember(member: Member) =
        invitationJPARepository.findByMember(MemberEntity.from(member)).map(InvitationEntity::toDomain)
```
이렇게 수정하면 코드가 더 깔끔해질 것 같습니다!

Reply:
람다로 한 번 감싸는 것보다 더욱 직관적으로 보입니다! 단순히 함수만 호출하는 경우라면, 메서드 참조 방식을 사용하는 것이 가독성 측면에서도 더욱 간결해보이네요 😄